src/util/report-migration.xml

```diff
@@ -39,7 +39,7 @@
           <summary>During the upgrade some fields have been removed. The records below have been automatically corrected.</summary>
           <ul>
             <t t-foreach="messages[category]" t-as="message">
-              <li><a t-att-href="'web?debug=1#view_type=form&amp;model=%s&amp;id=%s' % (message[0][0], message[0][1])" t-esc="message[0][2]" target="_blank"/></li>
+              <li><a t-att-href="'/web?debug=1#view_type=form&amp;model=%s&amp;id=%s' % (message[0][0], message[0][1])" t-esc="message[0][2]" target="_blank"/></li>
```

Comment:
I would instead use the `get_anchor_link_to_record` helper (which then need to be given in the `render_template` call).
```suggestion
              <li><t t-raw="get_anchor_link_to_record(*message[0])"/></li>
```